.packit.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# This script handles any custom processing of the spec file generated using the `post-upstream-clone`
+# action and gets used by the fix-spec-file action in .packit.yaml.
+
+set -eo pipefail
+
+REPO="gvisor-tap-vsock"
+SPEC=$REPO.spec
+
+# Get Version from HEAD
+# FIXME: Don't know where to fetch this from yet
+VERSION=0.0
```

Comment:
I would use `git describe` for this, but feels like getting a proper RPM version would be messy (if `git describe --exact HEAD` succeeds, then use this as the version number without the leading `v`, otherwise use `git describe --abbrev=0`, increment by 1, and append `git rev-parse --short HEAD`). Quite complicated now that I typed it :(

Reply:
@cfergeau `git describe --abbrev=0` works, thanks. Updated in https://github.com/containers/gvisor-tap-vsock/pull/206